cmd/verify_test.go

```diff
@@ -157,6 +159,7 @@ func TestCertify(t *testing.T) {
 
 		cmd.SetArgs([]string{
 			"-e", "is-helm-v3", // only consider a single check, perhaps more checks in the future
+			"-V", "4.9",
 			"-o", "yaml",
```

Comment:
Also, there isn't a failure check when the version can't be obtained nor given by the user.

Reply:
Currently, the tool exit with an error, and no report is produced. I don't think a check is necessary.